redisinsight/ui/src/components/database-overview/DatabaseOverview.tsx

```diff
@@ -166,16 +175,23 @@ const DatabaseOverview = (props: Props) => {
             { [styles.noModules]: !modules.visible?.length, [styles.RediStack]: isRediStack }
           )}
         >
-          {isRediStack
-            ? (
-              <DatabaseListModules
-                content={isRediStack ? RediStackLogo : undefined}
-                modules={modulesProps}
-                withoutStyles
-              />
-            )
-            : !!modules.visible?.length && (<DatabaseListModules dark inCircle modules={modules.visible} />)
-          }
+          {isRediStack ? (
```

Comment:
Cloud split logic to 2 conditional instead of Conditional Operator? like:
`{isRediStack && ....}`
`{!isRediStack && ....}`

Reply:
done